src/dmd/mars.d

```diff
@@ -2579,6 +2579,20 @@ bool parseCommandLine(const ref Strings arguments, const size_t argc, ref Param
             continue;
         }
     }
+
+    if (!params.vsafe && !params.noVsafe)
+    {
+        deprecation(
+            Loc.initial,
+            "-preview=dip1000 will become the default then deprecated.");
+        deprecationSupplemental(
+            Loc.initial,
+            "Please use -revert=dip1000 to maintain current behavior.");
+        deprecationSupplemental(
+            Loc.initial,
+            "https://github.com/dlang/DIPs/blob/master/DIPs/other/DIP1000.md");
```

Comment:
Ahem, this DIP is superseeded

Reply:
Oops. I guess the link should be [this](https://dlang.org/spec/memory-safe-d.html#scope-return-params) then?